### ISSUE TITLE
Middleman v4 support

### ISSUE THREAD

**Issue by alem0lars** [follmann/middleman-favicon-maker#34]

**Comment by crtvhd** [comment]
+1

**Comment by follmann** [comment]
Recently I don't have a lot of time to spend on this project. A pull request would be much appreciated. Thanks!

**Comment by crtvhd** [comment]
I'm using your fork @LMolr, thanks for addressing the v4 issue. However I'm getting another error which I don't know if it is from the extension itself or ImageMagick. Any idea?

```ruby
convert: unable to open image `/Users/martin/Development/Projects/wardanetwork-v2/build/assets/images/favicons/favicon-152x152.png': No such file or directory @ error/blob.c/OpenBlob/2709.
convert: WriteBlob Failed `/Users/martin/Development/Projects/wardanetwork-v2/build/assets/images/favicons/favicon-152x152.png' @ error/png.c/MagickPNGErrorHandler/1805.
      create  build/assets/images/favicons/favicon-152x152.png

```

**Comment by crtvhd** [comment]
@LMolr can you please do a pull request?
Much love!

**Comment by LMolr** [comment]
@crtvhd Aye, done in #35 :)

**Comment by follmann** [comment]
After taming some relative path issues, I have just released `middleman-favicon-maker (4.0.0.beta.1)`. Thanks!

**Comment by crtvhd** [comment]
Works fine! Thanks guys.

However stumbles across the error I mentioned above: 
My config looked like this:
```ruby
  activate :favicon_maker do |f|
    f.template_dir  = File.join(root, 'source/images/favicons')
    f.output_dir    = File.join(root, 'build/images/favicons')
    […]
```

Because my asset source folder ```source/images/favicons``` contained only files that are being ignored by the asset pipeline (prefixed with ```_filename.png```), middleman didn't create the destination folder ```build/images/favicons``` during build. That's the reason why ```middleman-favicon-maker``` failed during build. However as soon as I created the destination folder manually, everything worked fine. Keep in mind that your destination folder (```f.output_dir```) _has_ to exist before this gem is able to do it's magic.
